### PR TITLE
[l10n] Improve Ukrainian (uk-UA) locale

### PR DESCRIPTION
**Pull request by Neonin**
#3211

### CODE REVIEW

**Review by alexfauquette — APPROVED** [review]
Thanks for the update 👍

File: packages/grid/x-data-grid/src/locales/ukUA.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ const ukUAGrid: Partial<GridLocaleText> = {
   // Column menu text
   columnMenuLabel: 'Меню',
   columnMenuShowColumns: 'Показати стовпці',
-  // columnMenuManageColumns: 'Manage columns',
+  columnMenuManageColumns: 'Керування колонками',
```
Comment (cherniavskii) [inline packages/grid/x-data-grid/src/locales/ukUA.ts:106]:
```suggestion
  columnMenuManageColumns: 'Керування стовпцями',
```
Ref: https://learn.microsoft.com/uk-ua/power-apps/maker/data-platform/data-platform-manage-fields

File: packages/grid/x-data-grid/src/locales/ukUA.ts [hunk 2]
```diff
@@ -174,11 +174,11 @@ const ukUAGrid: Partial<GridLocaleText> = {
 
   // Aggregation
   aggregationMenuItemHeader: 'Агрегація',
-  // aggregationFunctionLabelSum: 'sum',
-  // aggregationFunctionLabelAvg: 'avg',
-  // aggregationFunctionLabelMin: 'min',
-  // aggregationFunctionLabelMax: 'max',
-  // aggregationFunctionLabelSize: 'size',
+  aggregationFunctionLabelSum: 'сумма',
```
Comment (cherniavskii) [inline packages/grid/x-data-grid/src/locales/ukUA.ts:177]:
```suggestion
  aggregationFunctionLabelSum: 'сума',
```

File: packages/grid/x-data-grid/src/locales/ukUA.ts [hunk 3]
```diff
@@ -174,11 +174,11 @@ const ukUAGrid: Partial<GridLocaleText> = {
...
+  aggregationFunctionLabelAvg: 'срзнач',
+  aggregationFunctionLabelMin: 'мін',
+  aggregationFunctionLabelMax: 'макс',
+  aggregationFunctionLabelSize: 'розмір',
```
Comment (cherniavskii) [inline packages/grid/x-data-grid/src/locales/ukUA.ts:181]:
```suggestion
  aggregationFunctionLabelAvg: 'сер',
  aggregationFunctionLabelMin: 'мін',
  aggregationFunctionLabelMax: 'макс',
  aggregationFunctionLabelSize: 'кількість',
```
Ref: Google Spreadsheet

**Review by cherniavskii — COMMENTED** [review]
Дякую!

**Pull request merged**